### PR TITLE
Fixes _recalculateColumnWidths method is called after children items of expended column loaded

### LINKED ISSUE #855: wheel can not work when mouse in the grid can not scroll page
wheel can not work when mouse in the grid can not scroll page

### PR DESCRIPTION
**Pull request by hongduc-phan**
Fixes #855

### CODE REVIEW

**Comment by hongduc-phan** [comment]
PR is not working. so I close it and continue to fix the bug

File: test/tree-toggle.html [hunk 1]
```diff
@@ -158,6 +169,32 @@
...
     });
+
+    describe('async recalculateWidth columns', () => {
+      let grid;
+    
+      beforeEach(() => {
+        grid = fixture('aysnc-tree-grid');
+      });
+    
+      it('should recalculate width columns when children nodes rendered', () => {
```
Comment (tomivirkki) [inline test/tree-toggle.html]: This test doesn't relate to tree-toggle so it shouldn't be in this file. Relocate to `column-auto-width`. Fix the description typos "should recalculate column widths when child items loaded"

File: test/tree-toggle.html [hunk 2]
```diff
@@ -32,6 +32,17 @@
     </template>
   </test-fixture>
 
+  <test-fixture id="aysnc-tree-grid">
+    <template>
+      <vaadin-grid style="width:600px;">
```
Comment (tomivirkki) [inline test/tree-toggle.html]: Unneeded inline style

File: test/tree-toggle.html [hunk 3]
```diff
@@ -32,6 +32,17 @@
...
+        <vaadin-grid-tree-column auto-width path="name"></vaadin-grid-tree-column>
+        <vaadin-grid-column auto-width>
```
Comment (tomivirkki) [inline test/tree-toggle.html]: Unneeded column

Comment (hongduc-phan) [reply test/tree-toggle.html]: done

File: test/tree-toggle.html (hunk 2 above)
Comment (hongduc-phan) [reply test/tree-toggle.html]: done

File: test/tree-toggle.html (hunk 1 above)
Comment (hongduc-phan) [reply test/tree-toggle.html]: moved it. thank you for correcting.

**Pull request merged**